src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -379,6 +380,12 @@ export function getPersonalisation(
   }
 }
 
+function literal(str: string) {
+  return `\`\`\`
+${str}
+\`\`\``
+}
```

Comment:
Do we need a trailing new line and/or Markdown `text` flavour?

```suggestion
function literal(str: string) {
  return `\`\`\`text\n${str}\n\`\`\``
}
```